src/main/java/wooteco/subway/admin/controller/StationController.java

```diff
@@ -30,12 +30,12 @@ public StationController(StationRepository stationRepository) {
 
     @GetMapping("/stations")
     public ResponseEntity<List<StationResponse>> showStations() {
-        return ResponseEntity.ok().body(StationResponse.listOf(stationRepository.findAll()));
+        return ResponseEntity.ok().body(StationResponse.listOf(stationService.findAll()));
     }
 
     @DeleteMapping("/stations/{id}")
     public ResponseEntity deleteStation(@PathVariable Long id) {
```

Comment:
```suggestion
    public ResponseEntity<Void> deleteStation(@PathVariable Long id) {
```

응답에 값이 없어도 제네릭에는 명시적으로 작성해주는 것이 좋지 않을까요? ㅎㅎ
큰 상관은 없어서 편하신대로 해주시면 됩니다 :)